tests/test_adapted_attributes.py

```diff
@@ -6,9 +6,11 @@
 from itertools import zip_longest
 from . import schema_adapted
 from .schema_adapted import Connectivity, Layout
-from . import PREFIX, S3_CONN_INFO
 
-SCHEMA_NAME = PREFIX + "_test_custom_datatype"
+
+@pytest.fixture
+def schema_name(prefix):
+    return prefix + "_test_custom_datatype"
```

Comment:
Switched this to a fixture, because it uses `prefix` which is now also a fixture.